studio/pages/account/me.tsx

```diff
@@ -77,6 +82,41 @@ const ProfileCard = observer(() => {
   )
 })
 
+const GithubProfile = observer(() => {
+  const { ui } = useStore()
+
+  return (
+    <Panel
+      title={[
+        <Typography.Title key="panel-title" level={5} className="mb-0">
+          Account Information
+        </Typography.Title>,
+      ]}
+    >
+      <Panel.Content>
+        <div className="space-y-2">
+          <Input
+            readOnly
+            disabled
+            label="Username"
+            layout="horizontal"
+            value=""
+            placeholder={ui.profile?.username ?? ''}
```

Comment:
i used placeholder just to give the feel that its not editable since it'll be grayed out
`value` for `Input` will be full white even if its disabled

@MildTomato perhaps we should gray out the input fields if disabled is true in the Ui library?

Reply:
Yep - i've added that in with new theming changes I've been doing. 
sorry, a lot of these smaller response style like focus, disabled etc have been done all at once with new styling system - will really try and get these changes out soon into beta release and can start applying them to studio.